srcpkgs/source-serif-pro/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'source-serif-pro'
+pkgname=source-serif-pro
+version=4.004
+revision=1
+wrksrc="source-serif-${version}R"
+depends="font-util"
+short_desc="Serif font family for user interface environments"
+maintainer="Nikolay Mukhachev <nikolay-mukhachev@yandex.ru>"
+license="OFL-1.1"
+homepage="https://adobe-fonts.github.io/source-serif/"
+distfiles="https://github.com/adobe-fonts/source-serif/archive/${version}R.tar.gz"
+checksum=dfb364735699cb830caad534cf7741234804d28e4b6fc5e4736b2401f6131aba
+font-dirs="/usr/share/fonts/OTF /usr/share/fonts/TTF"
+
+do_install() {
+	vmkdir usr/share/fonts/TTF
+	vmkdir usr/share/fonts/OTF
+	install -m644 ${wrksrc}/TTF/*.ttf ${DESTDIR}/usr/share/fonts/TTF
+	install -m644 ${wrksrc}/OTF/*.otf ${DESTDIR}/usr/share/fonts/OTF
```

Comment:
I think these two lines should use [`vinstall`](https://github.com/void-linux/void-packages/blob/master/Manual.md#global-functions) instead of `install`?

Reply:
Thanks for your advice.
I've fixed the template